park_info.clj

```diff
@@ -1,198 +1,197 @@
+    or at Libocká street, restaurant U Holečků, www.uholecku.cz,
+    or at Libocká studna, tel. +420 777 836 966,
+    alternatively at Petřiny"
+    :toilet true
+    :toilet_extra "Portable toilets on the premises, during summer season toilets are also available in the villa"
+    :POIs "Suitable for tourists and families with children, heritage trees, nature trail, natural attractions"
+    ;---Children and Sport---
+    :bike_path true
+    :skating false
+    :sports_field false
+    :sports_field_extra "nearby: Youth House (www.ddmp6.cz/kontakt/) & traffic playground"
+    :playground true
+    :playground_extra "2 playgrounds"
+    ;---Directions---
+    :public_transport "
+    stop: Nad Markétou - buses: #179, #184, #191, #510, 
+    stop: Libocká - buses: #179, #510, 
+    stop: Vypich - trams: #15, #22, #25, #57"
+    :GPS_coords "50°5′2.798″ N, 14°19′45.529″ E"
+    :parking true
+    :parking_extra "8 parking spots at Libocká gate, free parking in U Hvězdy street"
+    ;---Additional Information---
+    :dogs false
+    :path_type "Hardened soil, threshing floor"
+    :administered_by "Museum of Czech Literature, Prague City"
+    :opening_hours "park - year long, villa - (April 10:00-17:00, May-September 10:00-18:00, October 10:00-17:00, November-March CLOSED) "
+    ;---Map---
+    :map_url "https://www.praha.eu/public/bf/26/3e/184361_4_hvezda_mapa.pdf"
+})
+
+(def Kampa{
+    ;---Basic Info---
+    :food true
+    :food_extra "restaurant: Rybí Restaurace,
+    café: Mlýnská (tel. +420 608 444 490)
+    restaurant/café : Sovovy Mlýny"
+    :toilet true
+    :toilet_extra nil
+    :POIs "View of the river, Sovovy Mlýny gallery, Čertovka, Charles Bridge, Werich House, heritage trees"
+    ;---Children and Sport---
+    :bike_path true
+    :skating true
+    :sports_field false
+    :sports_field_extra "no specialized sports facilities, but grass-covered segments available for sports"
+    :playground true
+    :playground_extra "yes, in the vicinity: Community and Maternal centre Kampa"
+    ;---Directions---
+    :public_transport "
+    stop: Helichova - tram:s #12, #20, #20, #22, #23, #57 
+    stop: Újezd - trams: #6, #9, #22, #23, #57, #58, #59"
+    :GPS_coords "50°5′16.87″ N, 14°24′14.58″ E"
+    :parking false
+    :parking_extra nil
+    ;---Additional Information---
+    :dogs nil
+    :path_type "asphalt"
+    :administered_by "Municipality Prague 1, Prague City"
+    :opening_hours "Open year-round"
+    ;---Map---
+    :map_url "https://www.praha.eu/public/7c/a6/ed/184308_4_kampa_mapa.pdf"       
+})
+
+(def Kinskeho_zahrada{
+    ;---Basic Info---
+    :food true
+    :food_extra "café in Musaion museum of natural sciences"
+    :toilet true
+    :toilet_extra nil
+    :POIs "Enjoyable location for a walk with children, Quiet and relaxing space in the center of town, Musaion Museum of Natural Sciences, decorative ponds, natural attraction - Petřín rock formations"
```

Comment:
'Suitable for' in the beginning.

Reply:
These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.